index.js

```diff
@@ -1,10 +1,40 @@
 'use strict'
-
 const axeCore = require('axe-core')
 const merge = require('lodash.merge')
 const chalk = require('chalk')
 const { printReceived, matcherHint } = require('jest-matcher-utils')
 
+/**
+ * Converts a HTML string or HTML element to a mounted HTML element.
+ * @param {element | string} a HTML element or a HTML string
+ * @returns {[element, function]} a HTML element and a function to restore the document

```

Comment:
`element` is not a valid JSDoc type. `Element` is a valid type, though.

Reply:
Ah, yes, sorry. I have reverted this.